### PR TITLE
VACMS-17436: Update links for manage your health care section

### PR DESCRIPTION
**Pull request by chriskim2311**
## Summary
For this PR we are adding 2 new links under the Manage your health care section.

Download VA Health Chat and Connect to VA Care.

Also we changed the header as well to an h2 to fix an a11y defect.

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17436

## Testing done
Tested locally at `/greater-los-angeles-health-care/`

Verified links are working and redirecting to correct address.

## Screenshots
VA_Greater_Los_Angeles_Health_Care___Veterans_Affairs

## What areas of the site does it impact?

VA Health Care system pages

### Add VA Health Chat
- [x] add 'chat' USWDS v3 icon 
- [x] include `aria-hidden="true"` on icon
- [x] add link text "Download VA Health Chat" with a link [https://mobile.va.gov/app/va-health-chat]
### Add VA Health connect 
- [x] add 'phone' USWDS v3 icon 
- [x] include `aria-hidden="true"` on icon
- [x] add link text "Connect to VA care" with a link https://www.va.gov/health/connect-to-va-care/index.asp

### Additional Acceptance Criteria
- [ ] all existing links remain
- [ ] new links should appear below existing links
- [ ] Change header to h2 (currently h3) 

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### CODE REVIEW

**Comment by laflannery** [comment]
@chriskim2311 `aria-hidden` was added on the `va-link` which is not correct this is for the icon only and in fact is already being done automatically because we are using `va-icon`. Can you remove this from the links?

**Review by randimays — COMMENTED** [review]
There's an additional AC in the ticket about keeping these links "below existing links" but on mobile they are nested within the other links per the structure:

Screenshot 2024-07-12 at 9 24 03 AM

Screenshot 2024-07-12 at 9 22 51 AM

Could we confirm with Michelle & design what the behavior should be on mobile? (If we need to reorder things, we can use flex / order to achieve this simply, I think).

**Comment by laflannery** [comment]
@randimays Nice catch! Unfortunately though I believe flex order is the property that makes the tab order and the visual order off and confusing (If I'm thinking of the right thing?) However I had a different question for Design I pinged Amanda on [in Slack](https://dsva.slack.com/archives/C0FQSS30V/p1720788999624339) and I will add this to that thread

File: src/site/assets/sass/style.scss [hunk 1]
```diff
@@ -54,6 +54,18 @@
...
 }
 
+.health-online-desktop-link {
+  @media (max-width: 767px) {
+    display: none !important;
+  }
+}
+
+.health-online-mobile-link {
+  @media (min-width: 767px) {
```
Comment (randimays) [inline src/site/assets/sass/style.scss:64]: This should be `min-width: 768px`

**Review by laflannery — APPROVED** [review]
Looks good!

**Comment by chriskim2311** [comment]
@randimays Can you review this PR thanks!

File: src/site/assets/sass/style.scss [hunk 2]
```diff
@@ -54,6 +54,18 @@
   word-wrap: break-word;
 }
 
+.health-online-desktop-link {
+  @media (max-width: 768px) {
```
Comment (randimays) [inline src/site/assets/sass/style.scss:58]: This should still be 767. There shouldn't be overlap between max-width and min-width queries for the same element.

Comment (chriskim2311) [reply src/site/assets/sass/style.scss:58]: Ahh sorry Friday brain. I'm making the change now

**Review by randimays — APPROVED** [review]

**Review by IGallupSoCo — APPROVED** [review]
LGTM

**Pull request merged**